README.md

```diff
@@ -0,0 +1,83 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# FPS Games Global CON 2022
+
+> The global event of first person shooter games
+
+Microverse Capstone Project - HTML/CSS & JavaScript Conference page
+
+
+## Live Server
+
+In the following link, you can view [FPSCON 2022](https://pepedropo1.github.io/Portfolio/).
+
+
+## Built With
+
+- HTML;
+- CSS;
+- JavaScript;
+- Fontawesome 4.
+
+
```

Comment:
- [ ] One thing left to be done is creating a presentation video using [Loom](https://www.loom.com/). I recommend creating a less than 5 mins short video about your website, for more information please read [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) 👋🍀